backend/battle/endpoints.py

```diff
@@ -0,0 +1,40 @@
+from django.db.models import Q
+
+from rest_framework import generics
+from rest_framework.permissions import IsAuthenticated
+
+from battle.permissions import IsInBattle
+from battle.models import Battle
+from battle.serializers import BattleSerializer, BattleCreateSerializer, TeamCreateSerializer
+
+
+class BattletList(generics.ListCreateAPIView):
+    serializer_class = BattleSerializer
+    permission_classes = (IsAuthenticated,)
```

Comment:
All other endpoints are using lists, let's keep the consistency by using lists here too

Reply:
great!